drivers/console/uart_console.c

```diff
@@ -599,6 +599,11 @@ static int uart_console_init(const struct device *arg)
 	/* Claim console device */
 	uart_console_dev = device_get_binding(CONFIG_UART_CONSOLE_ON_DEV_NAME);
 
+	if (!uart_console_dev) {
+		printk("Config UART console port was not found!\n");
```

Comment:
Not sure that this will work, the printk handler is only installed  after this check.